client/reader/following/intro.jsx

```diff
@@ -3,22 +3,20 @@
  */
 import React from 'react';
 import { localize } from 'i18n-calypso';
-import { connect } from 'react-redux';
```

Comment:
will we still be using `props.requestGraduate` etc.?
I'd prefer we removed those lines even if adding them back in shortly